src/PKSim.Core/Snapshots/IdentificationParameter.cs

```diff
@@ -0,0 +1,12 @@
+﻿using OSPSuite.Core.Domain;
+
+namespace PKSim.Core.Snapshots
+{
+   public class IdentificationParameter : ParameterContainerSnapshotBase
+   {
+      public Scalings Scaling { get; set; }
+      public bool? UseAsFactor { get; set; }
+      public bool? IsFixed { get; set; }
+      public string[] LinkedParameters { get; set; }
```

Comment:
Why are `UseAsFactor  `and `IsFixed `defined as nullable?

Reply:
The way the snapshot engine works is that it allows to NOT specify default value (since the goal was to only write what was set by the user). Typically, UseAsFactor and IsFixed are not used (in probably 90% of the cases and even more). No need to save a value. When deserializing, missing value is interpreted as false

So instead of true or false, we are using the concept of truthy and falsy (falsy = false, null in that case)

Reply:
The question for me is what should happen if we change a default.
In case of e.g. a physiological parameter the new default should be used.

But here I'm not quite sure. If we change the default for *UseAsFactor*: should the PI recreated from snapshot use this new default or the previous value? (and I would vote for the previous one)

Anyway, default values for the settings above will probably never be changed, so we can keep it as is.

Reply:
@Yuri05 I believe this is the correct behavior. If we change the default!, we expect the snapshot to run with the default. No result is saved so the PI would result in the same behavior as IF the snapshot was created from scratch. 
Now to be honest, this does not really matter for those guys and we can do whatever . But this logic of sticking with App default is present everywhere and just not here as per requirement. (Only save, what was set by the user)